<?xml version="1.0"?>
<launch>
      <param name="robot_description" command="$(find xacro)/xacro.py '$(find pr2_description)/robots/pr2_no_kinect.urdf.xacro'" />

      <param name="robot_description_sot" command="$(find xacro)/xacro.py '$(find sot_robot)/robot/urdf/pr2/pr2_sot.urdf.xacro'" />

    <arg name="gui" default="False" />
    <param name="use_gui" value="$(arg gui)"/>
    <param name="robot_base_link" value="base_footprint"/>
    <!--<param name="ref_link" value="odom_combined"/>-->

    <arg name="frame" value="odom"/>
    <arg name="child_frame" value= "base_footprint"/>
<param name="ref_link" value="odom_combined"/>


    <node name="robot_pose_publisher" pkg="dynamic_graph_bridge" type="robot_pose_publisher" args="_frame:=$(arg frame) _child_frame:=$(arg child_frame)">
        <env name="ROS_NAMESPACE" value="" />
    </node>

     <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" >
	<env name="ROS_NAMESPACE" value=""/>
    </node>

</launch>

